worker/index.test.js

```diff
@@ -1,25 +1,41 @@
 const { unstable_dev } = require("wrangler");
+const { rawHtmlResponse, loginForm } = require("./functions/response");
+require('isomorphic-fetch');
```

Comment:
> When I save this file in my editor eslint automatically insterts `""` instead `''`

please use quotes `" "` instead `''` to keep it consistent with rest of the file.